### DISCUSSION TITLE
Quickly wrap any elements with a container or div for better structure control

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#24311] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, managing multiple widgets in Elementor's Structure Panel can be cumbersome, especially when you want to group them together. The lack of a streamlined method for wrapping selected widgets in a container makes the organization of complex layouts less efficient. This feature aims to address this issue and enhance the user experience by providing a convenient way to group widgets within a container directly from the Structure Panel.

### Describe the solution you'd like

Introduce a new feature that allows users to select one or more widgets in the Structure Panel and, using the context menu, choose the option to "Wrap in Container." Upon selecting this option, Elementor should automatically generate a new container, and the chosen widgets should be wrapped as children within this container. This feature would provide a more intuitive and time-saving method for organizing and managing widgets in the Structure Panel.

The menu should look like this
![option](https://github.com/elementor/elementor/assets/99708427/b0aa6622-e76d-4812-8a73-0e3cd1789642)

After wrapping
![after wrapping](https://github.com/elementor/elementor/assets/99708427/dd201d1e-b424-4ab9-b1bd-9b8c7d8153fb)
ing, It should look like this

### Describe alternatives you've considered

- Manually dragging and dropping selected widgets into a container on the canvas, which can be less efficient and disrupt the layout.
- Creating a container separately and then manually assigning widgets to it, involving additional steps and a higher likelihood of errors.

### Additional context

- This feature aligns with the goal of Elementor to enhance functionality for a more user-friendly and efficient website development experience.
- It significantly improves the workflow for developers and designers, making it easier to manage and structure complex layouts.

**Comment by ayoubkhan558** [comment]
+1

**Comment by ryanmish** [comment]
I'm surprised to see this hasn't been added after over a year.

**Comment by vovkasolovev** [comment]
I agree. Very often I want to press something common like _Ctrl+G_ to **wrap the selected widget in a container**. Since Elementor now allows you to select multiple widgets, it would be great if they all went into the container if the widgets are siblings.

And of course, we need a reverse shortcut: to remove from the container if you select the container. It is also often desirable to use an action familiar from other design applications, such as _Ctrl+Shift+G_.
